### PR TITLE
initial kuttl setup for KUDO Kafka MWT

### PR DESCRIPTION
**Pull request by zmalik**
This PR adds initial setup of MWT using KUTTL

### CODE REVIEW

File: benchmarks/mwt/workload-large/00-mwt-workload/00-install-workload.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - command: kubectl apply -f ./workload.yaml
```
Comment (kensipe) [inline benchmarks/mwt/workload-large/00-mwt-workload/00-install-workload.yaml:4]: this is another strange apply... we can use the `apply:` property... or place the file in this file.

File: benchmarks/mwt/workload-small/00-small-workload/00-install-workload.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - command: kubectl apply -f ./workload.yaml
```
Comment (kensipe) [inline benchmarks/mwt/workload-small/00-small-workload/00-install-workload.yaml:4]: apply?

File: benchmarks/mwt/setup/01-zookeeper-install/02-install-zookeeper.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - command: kubectl kudo install zookeeper --namespace kafka-mwt
```
Comment (kensipe) [inline benchmarks/mwt/setup/01-zookeeper-install/02-install-zookeeper.yaml:4]: might be worth `  kubectl kudo install zookeeper --namespace kafka-mwt --wait`

File: benchmarks/mwt/setup/02-kafka-install/02-install-kafka.yaml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - command: kubectl kudo install kafka --namespace kafka-mwt -P ${KAFKA_PARAMETER_FILE}.yaml
```
Comment (kensipe) [inline benchmarks/mwt/setup/02-kafka-install/02-install-kafka.yaml:4]:
```suggestion
  - command: kubectl kudo install kafka --namespace kafka-mwt -P ${KAFKA_PARAMETER_FILE}.yaml --wait
```

File: benchmarks/mwt/setup/03-dashboard-install/00-install-grafana-dashboard.yaml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - command: kubectl apply -f ./dashboard-cm.yaml
```
Comment (kensipe) [inline benchmarks/mwt/setup/03-dashboard-install/00-install-grafana-dashboard.yaml:4]: apply?

File: benchmarks/mwt/verify-workload/00-install-utils-pod.yaml [hunk 6]
```diff
@@ -0,0 +1,5 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - command: kubectl apply -f ./busybox.yaml
+    ignoredFailure: true
```
[the remark this answers is not in the document]
Comment (zmalik) [reply benchmarks/mwt/verify-workload/00-install-utils-pod.yaml:5]: having busybox.yaml content here was deleting it right after test was finished. With this Apply pattern it keeps them running.

File: benchmarks/mwt/workload-large/00-mwt-workload/00-install-workload.yaml (hunk 1 above)
Comment (zmalik) [reply benchmarks/mwt/workload-large/00-mwt-workload/00-install-workload.yaml:4]: same, workload was deleted right after

File: benchmarks/mwt/README.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+     
+## Verification
+
+run the next command to setup the utils pod we will use for verification
+
+```
+kubectl kuttl verify-workload/
+```
+
+Now that kuttl confirms that utils pod is up and ready, we can exec inside the pod to run verifications
```
[the remark this answers is not in the document]
Comment (zmalik) [reply benchmarks/mwt/README.md:70]: let me expand that part

File: benchmarks/mwt/setup/01-zookeeper-install/02-install-zookeeper.yaml (hunk 3 above)
Comment (zmalik) [reply benchmarks/mwt/setup/01-zookeeper-install/02-install-zookeeper.yaml]:
```
"kubectl kudo install kafka --namespace kafka-mwt -P ${KAFKA_PARAMETER_FILE}.yaml --wait" exceeded 30 sec timeout
```
adding `--wait` resulted in timeout errors of yesterday MWT, I will remove the wait as this should be done through assert step

**Comment by zmalik** [comment]
updated the README.md of the mwt directory as it was missing what we actually are trying to achieve with this automation

**Pull request merged**